sympy/combinatorics/fp_groups.py

```diff
@@ -1177,21 +1184,25 @@ def _simplification_technique_1(rels):
 ###############################################################################
 
 # Pg 175 [1]
-def define_schreier_generators(C, rel=False):
+def define_schreier_generators(C, homomorphism=False):
     '''
     Arguments:
     C -- Coset table.
-    rel -- When set to True, this returns a relation between subgroup generators
-           and the elements of the parent group.
+    homomorphism -- When set to True, this returns a relation between subgroup generators
+                  and the elements of the parent group.
```

Comment:
"A relation" is too ambiguous. What sort of a relation? Also, saying "return" rather than "this returns" would be more consistent with docstring conventions (even if this isn't the first line).

So maybe "Return a dictionary containing the images of the presentation generators in the original group"